masonry/src/contexts.rs

```diff
@@ -716,6 +717,13 @@ impl EventCtx<'_> {
         self.is_handled
     }
 
+    /// The widget originally targeted by the event.
+    ///
+    /// This will be different from [`widget_id`](Self::widget_id) during event bubbling.
```

Comment:
Is it actually meaningful for accessibility events to bubble?

Reply:
I *think* it is? Though we don't really use bubbling anywhere for now.

> Every implementation seems to check if the event targets self

I think most of those checks are vestigial, from the initial implementation where accessibility events were sent to the entire tree (with the bloom filter to cut some subtrees off).